src/java.base/share/classes/java/util/ResourceBundle.java

```diff
@@ -3729,7 +3729,7 @@ private static String toPackageName(String bundleName) {
 
     }
 
-    private static final boolean TRACE_ON = Boolean.valueOf(
+    private static final boolean TRACE_ON = Boolean.parseBoolean(
```

Comment:
I think for `Boolean` it doesn't make significant difference, as both `Boolean.TRUE` and `Boolean.FALSE` are cached and initialized immediately along with the class itself:
```java
public static Boolean valueOf(String s) {
  return parseBoolean(s) ? TRUE : FALSE;
}
```

Reply:
Yes, there is no allocation for booleans. But still 2 redundant method calls in interpreter mode (valueOf+booleanValue)